spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,66 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    it 'copy template with the same content' do
+      test_ot_1 = FactoryGirl.create(:orchestration_template_cfn)
+
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      run_post(orchestration_templates_url(test_ot_1.id), gen_request(:copy))
+      expect(response).to have_http_status(:bad_request)
+    end
+
+    it 'copy single template with the different content' do
+      test_ot_1 = FactoryGirl.create(:orchestration_template_cfn)
+      test_content_1 = "{ 'Description': 'Test content 1' }\n"
+
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      expected = {
+        'content' => test_content_1
```

Comment:
I will add the verification for id, but adding verification for the href a little bit problematic, FactorGirl  OrchestrationEntity does not have the href.
I believe to check only id's good enough for test purpose.
What do you think?